app/src/components/home/Home.test.js

```diff
@@ -59,6 +59,7 @@ it('submits a valid query', () => {
 
 it('tries to submit an empty query', () => {
   const SEARCH_STR = '';
+  const NOTIFY_MSG = 'Minimum length of Search word must be 3 characters.';
```

Comment:
Is this constantized elsewhere in the project? if so, try to use that here

if not, it probably should be... nothing less fun than tweaking the string in the other place and then have your test fail here and not notice why for a while since the strings look so similar... trust me, i've been there :)